apps/docs/pages/guides/platform/going-into-prod.mdx

```diff
@@ -60,18 +60,37 @@ After developing your project and deciding it's Production Ready, you should run
-| Signup confirmation request                      | `/auth/v1/signup`                                              | Last request             | Defaults to 60 seconds window before a new request is allowed. Is customizable.                                                            |
-| Password Reset Request                           | `/auth/v1/recover`                                             | Last request             | Defaults to 60 seconds window before a new request is allowed. Is customizable.                                                            |
-| Verification requests                            | `/auth/v1/verify`                                              | IP Address               | 360 requests per hour (with bursts up to 30 requests)                                                                                      |
-| Token refresh requests                           | `/auth/v1/token`                                               | IP Address               | 1800 requests per hour (with bursts up to 30 requests)                                                                                     |
-| Create or Verify an MFA challenge                | `/auth/v1/factors/:id/challenge` `/auth/v1/factors/:id/verify` | IP Address               | 15 requests per minute (with bursts up to 30 requests)                                                                                     |
+Auth rates can be configured for your project in the [rate limits settings page](/dashboard/project/_/auth/rate-limits).
+
+#### Rate limits applied for sum of requests.
+
+The following are limits that are applied when the sum of requests exceeds the limit within a `1hr` window.
+
+| Path                                                           | Description                                                                                                                                                                                                             |
+| -------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `/auth/v1/signup`<br/> `/auth/v1/recover`<br/> `/auth/v1/user` | All endpoints that send emails. <br/> Defaults to `30 emails per hour`. <br/>Updated to `4 emails per hour` on 14th July 2023. Triggered when the sum of email send requests exceeds the set limit per hour. <br/><br/> |
+| `/auth/v1/otp`                                                 | All endpoints that send One-Time-Passwords (OTP). <br/>Defaults to `30 OTPs per hour`. <br/>Triggered when the sum of OTP requests exceeds the set limit per hour.<br/><br/>                                            |
+
+#### Limited by last sent request
+
+In addition, all of the following paths are limited by a `60s` window limit.
```

Comment:
I can see how this gets confusing, because if you want to figure out why `/auth/v1/otp` is rate-limited, for example, you might look at the section above and determine that it's limited by requests per hour. But in fact it's also limited by time since last request.

I don't know if you want to reorganize this again, but maybe we could have a summary table at the top that goes:

```
## Default rate limits

You can change the defaults ....

| Path | Number of requests in time window | Time since last request | Number of requests in time window per IP |
| - | - | - | - |
| `/auth/v1/otp` | 30 per hour | 60 s | - |
```

And then the path could link to a description section below (could be free-form or could use the table format here)